testing/tests/DevExpress.ui.widgets/toolbar.tests.js

```diff
@@ -1724,6 +1724,27 @@ QUnit.module('adaptivity without hiding in menu', {
     });
 });
 
+QUnit.module('Toolbar disposing', () => {
+    QUnit.test('_dimensionChanged skipped if widget is disposed (T1147410)', function(assert) {
+        const origIsMaterial = themes.isMaterial;
+        themes.isMaterial = () => true;
+
+        const toolbar = $('#toolbar').dxToolbar().dxToolbar('instance');
+
+        toolbar.dispose();
+
+        try {
+            toolbar.option('width', 250);
+        } catch (e) {
+            assert.ok(false, e);
+        } finally {
+            assert.ok(true, 'the exception is not thrown');
+        }
+
+        themes.isMaterial = origIsMaterial;
```

Comment:
jfyi: if an error occurs this code will not be run. We should always use `try/finally` for such restoring or move it to a module configuration. However, let's refactor the test to not use a material theme at all.